backend/src/entities/Meal.ts

```diff
@@ -0,0 +1,17 @@
+export class Meal {
+    id: Number;
+    name: String;
+    description: String | undefined;
```

Comment:
Koristis String i Number tipove, sta nije bas dobro jer to oznacava tip globalnog objekta, tj da se instancira kao new Number()
Triba koristit lowercase sta je za primitivne tipove a to je sta nama tu zapravo i je
Plus, za bolju citljivost netriba ti koristit -> | undefined, nego sintaksu opcionalnog parametra -> parametar?: string;

Znaci ovako bi to izgledalo: export class Meal {
    id: number;
    name: string;
    description?: string;
    price: number;
    image?: string;
    mealCategoryId: number;

    constructor(id: number, name: string, mealCategoryId: number, price: number, description?: string, image?: string) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.price = price;
        this.image = image;
        this.mealCategoryId = mealCategoryId;
    }
}